rebasehelper/specfile.py

```diff
@@ -1110,6 +1113,24 @@ def get_prep_section(self):
                 result.append(prep.pop(0))
         return result
 
+    def get_main_files_section(self):
+        """Finds the exact name of the main files section.
+
+        Returns:
+            str: Name of the main files section.
+
+        """
+        parser = SilentArgumentParser()
+        parser.add_argument('-n', default=None)
+        parser.add_argument('-f')
+        parser.add_argument('subpackage', nargs='?', default=None)
+
+        for sec_name in self.spec_content.sections:
+            if sec_name.startswith('%files'):
+                ns, _ = parser.parse_known_args(sec_name.split(' ')[1:])
```

Comment:
Same note about `split()` separator argument as above, but I think it's safer to use `shlex.split()` here. Although e.g. arguments containing whitespaces are probably not allowed in a spec file, it's better to be safe than sorry.